src/find-options/FindOneOptions.ts

```diff
@@ -111,4 +111,9 @@ export interface FindOneOptions<Entity = any> {
      * If this is set to true, SELECT query in a `find` method will be executed in a transaction.
      */
     transaction?: boolean
+
+    /**
+     * A timestamp to get datasets from temporal tables.
+     */
+    timestamp?: Date
```

Comment:
`timestamp` is too generic name. Maybe something like `temporaryTableTimestamp` or even something better.

Reply:
I change this property to `validAt`.